daemon/create.go

```diff
@@ -16,6 +17,13 @@ func (daemon *Daemon) ContainerCreate(name string, config *runconfig.Config, hos
 		return "", warnings, err
 	}
 
+	// The check for a valid workdir path is made on the server rather than in the
+	// client. This is because we don't know the type of path (Linux or Windows)
+	// to validate on the client.
+	if config.WorkingDir != "" && !filepath.IsAbs(config.WorkingDir) {
```

Comment:
@jhowardmsft heh I probably haven't reviewed thoroughly back then. :joy: I think runconfig.Parse is essentially where the `config` is deserialized from remote API request. So it'll be still a common place to do it as far as I can tell, also logically it makes sense to have it where the rest of the checks are, right?